blueocean-dashboard/src/main/js/creation/CreatePipeline.js

```diff
@@ -34,6 +35,11 @@ export default class CreatePipeline extends React.Component {
         }
     }
 
+    _isCreationDisabled() {
+        const user = User.current();
+        return Security.isSecurityEnabled() && user && !user.permissions.pipeline.create();
```

Comment:
This logic was being used in two places, so I refactored into a util in d1dd429. Let me know if you think it's an improvement.

Some of our security checks are using sorta slightly different approaches in terms of logic, so I am open to refining this stuff a bit.